readthedocsext/theme/templates/acl/shared_password_required.html

```diff
@@ -0,0 +1,28 @@
+{% extends "proxito/base.html" %}
+{% load trans blocktrans from i18n %}
+{% load crispy from crispy_forms_tags %}
+
+{% block title %}
+  {% trans "Authentication required" %}
+{% endblock title %}
+
+{% block content %}
+  <h1>{% trans "Authentication required" %}</h1>
+
+  <p>
+    {% blocktrans with project_name=project.name %}
+    You need special requirements to access "{{ project_name }}" project documentation.
+    Please, provide a valid password to see the docs.
+    {% endblocktrans %}
+  </p>
+
+  <form class="ui form" method="post">
+    {% csrf_token %}
+    {{ form|crispy }}
+    <button class="ui primary button" type="submit">{% trans "Log in" %}</button>
+  </form>
+
+  <p>
+    {% blocktrans %}Already have an account? Then please <a href="//{{ PRODUCTION_DOMAIN }}/accounts/login/">log in</a>.{% endblocktrans %}
```

Comment:
When using blocktrans, make sure to use a context variable instead of globals. It's a small change, but helps isolate translation changes from code changes:

```suggestion
    {% blocktrans with domain=PRODUCTION_DOMAIN %}Already have an account? Then please <a href="//{{ domain }}/accounts/login/">log in</a>.{% endblocktrans %}  
```

And maybe it's also worth consistently generating the path with `url`? djlint normally yells about this if the URL is relative

Reply:
> And maybe it's also worth consistently generating the path with url?

We don't have access to the URLs from the main dashboard from proxito.